### ISSUE TITLE
new ansistrano-deploy role repository address

### ISSUE THREAD

**Issue by DjeDt** [cbrunnkvist/ansistrano-symfony-deploy#22]
Hi, the ansistrano-deploy role is now here 
```
https://github.com/ansistrano/deploy
https://galaxy.ansible.com/ansistrano/ansistrano-deploy
```
> ansible-galaxy install --force ansistrano.ansistrano_deploy

in order to be able to use ansistrano-symfony-deploy again it would be necessary to update these files :
```
meta/main.yml
.travis.yml
README.md
```

I tried to create a branch in order to make a pull request but i got an access denied when i pushed it :(

**Comment by ricardclau** [comment]
The roles are now at ansistrano.deploy and ansistrano.rollback

Sorry about this... I am sure this role is also affected as it contains hyphens

**Comment by scuben** [comment]
Thank you @DjeDt for bringing this up. To create a pull request you need to do the changes in a fork of this repository.

**Comment by DjeDt** [comment]
Ok, the pull request is made, i hope it's ok !

**Comment by scuben** [comment]
Thank you @DjeDt for creating the PR and thank you @ricardclau for mentioning the renaming. I just released v1.3.0 which resolves the issue.
